config/evss/mock_letters_response.yml.example

```diff
@@ -1,4 +1,65 @@
 ---
+# default, edit but do not remove
+default:
+  :get_letters:
+    :body:
+      letter_destination:
+        address_line1: 2476 MAIN STREET
+        address_line2: 'STE # 12'
+        address_line3: West
+        city: RESTON
+        country: US
+        foreign_code: '865'
+        full_name: MARK WEBB
+        state: VA
+        zip_code: '12345'
+      letters:
+      - letter_name: Commissary Letter
+        letter_type: commissary
+      - letter_name: Proof of Service Letter
+        letter_type: proof_of_service
+      - letter_name: Proof of Creditable Prescription Drug Coverage Letter
+        letter_type: medicare_partd
+      - letter_name: Proof of Minimum Essential Coverage Letter
+        letter_type: minimum_essential_coverage
+      - letter_name: Service Verification Letter
+        letter_type: service_verification
+      - letter_name: Civil Service Preference Letter
+        letter_type: civil_service
+      - letter_name: Benefit Summary Letter
+        letter_type: benefit_summary
+      - letter_name: Benefit Verification Letter
+        letter_type: benefit_verification
+    :status: 200
+
+  :get_letter_beneficiary:
+    :body:
+      benefit_information:
+        has_non_service_connected_pension: false
+        has_service_connected_disabilities: true
+        has_survivors_indemnity_compensation_award: false
+        has_survivors_pension_award: false
+        monthly_award_amount: 123.0
+        service_connected_percentage: 2
```

Comment:
Can we add `award_effective_date: '1965-01-01T05:00:00.000+0000'` here too? Here and in the spec I guess. Thanks!